app/views/layouts/_navbar.html.erb

```diff
@@ -4,7 +4,7 @@
     <span class="navbar-toggler-icon"></span>
   </button>
   <div class="collapse navbar-collapse" id="navbarNav">
-    <ul class="navbar-nav">
+    <ul class="navbar-nav mr-auto">
```

Comment:
What does the mr-auto do?

Reply:
margin right auto, you can find out more here! https://getbootstrap.com/docs/4.3/utilities/spacing/